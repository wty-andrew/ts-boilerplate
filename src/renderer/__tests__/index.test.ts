import '../index'

test('render without crashing', () => {
  expect(document.body).toMatchSnapshot()
})
