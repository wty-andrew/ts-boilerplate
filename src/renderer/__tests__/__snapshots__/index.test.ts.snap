// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render without crashing 1`] = `
<body>
  Hello World
</body>
`;
